detox/local-cli/detox-build.js

```diff
@@ -2,14 +2,17 @@
 
 const _ = require('lodash');
 const program = require('commander');
+const {getConfigurationFile, getDefaultConfigurationFile} = require('../src/utils/getConfigurationFile');
 const path = require('path');
 const cp = require('child_process');
 program.description(`[convenience method] run the command defined in 'configuration.build'`)
-       .option('-c, --configuration [device configuration]', 'Select a device configuration from your defined configurations,'
-                                                             + 'if not supplied, and there\'s only one configuration, detox will default to it')
+  .option('--config-path [configPath]',
+    'Select a device config-file path, if not supplied, detox will default to the package.json, and if not found there, detox will fallback to .detoxrc')
```

Comment:
Currently the logic I made was `config-file -> package.json -> detoxrc`
You're suggesting to change it to
`config-file ->  detoxrc -> package.json` ?

Reply:
@noomorph, the order you suggest seems reasonable, seems like it follows the standard (AFAIK), [Babel for example](https://babeljs.io/docs/en/config-files).
I agree with your statement, it will probably (hopefully) won't break anything.

Reply:
@oreporan, go for it.